hack/pre-release.sh

```diff
@@ -105,8 +116,7 @@ update_OCP_version() {
   echo "using OCP version $version"
 
   # Increment the version by using the stream_var variable to select which column to increment 
-  updated_version=$(echo $version |
-    awk -F. -v OFS=. -v stream_var="$stream_var" '{$stream_var += 1 ; print}')
+  updated_version=$(increment_version $wmco_version $stream)
```

Comment:
I am confused here. Why are you updating the WMCO version in the `update_OCP_version()`.